hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoDeleteAllowanceSuite.java

```diff
@@ -504,28 +509,30 @@ private HapiSpec exceedsTransactionLimit() {
                                         List.of(
                                                 ByteString.copyFromUtf8("a"),
                                                 ByteString.copyFromUtf8("b"),
-                                                ByteString.copyFromUtf8("c")))
+                                                ByteString.copyFromUtf8("c"),
+                                                ByteString.copyFromUtf8("d"),
+                                                ByteString.copyFromUtf8("e")))
                                 .via("nftTokenMint"),
                         mintToken(token, 500L).via("tokenMint"),
-                        cryptoTransfer(movingUnique(nft, 1L, 2L, 3L).between(TOKEN_TREASURY, owner)))
+                        cryptoTransfer(movingUnique(nft, 1L, 2L, 3L, 4L, 5L).between(TOKEN_TREASURY, owner)))
                 .when(
                         cryptoApproveAllowance()
                                 .payingWith(owner)
                                 .addNftAllowance(owner, nft, spender, false, List.of(1L, 2L)),
                         cryptoDeleteAllowance()
                                 .payingWith(owner)
-                                .addNftDeleteAllowance(owner, nft, List.of(1L, 2L, 3L, 3L, 3L))
-                                .hasPrecheck(MAX_ALLOWANCES_EXCEEDED),
+                                .addNftDeleteAllowance(owner, nft, List.of(1L, 1L, 1L, 1L, 1L))
+                                .hasPrecheck(OK),
```

Comment:
Usually I wouldn't change the test conditions, but I think this change makes sense. I'm guessing this test wasn't passing before because the code was counting all the duplicate nft IDs (i.e. it counted ID 3 three separate times). The new implementation checks for duplicates and doesn't add them to the total, so I changed this expected precheck status to OK. In other words:
`.addNftDeleteAllowance(owner, nft, List.of(1L, 1L, 1L, 1L, 1L))` is the equivalent of `.addNftDeleteAllowance(owner, nft, List.of(1L))`

Reply:
hm, I would ask @Neeharika-Sompalli to double check this.

Reply:
If we are making it a set, then the implementation is wrong. We should comply with mono-service. We should not be changing this error code.

Reply:
This seems like a bug in mono service code, but I'll go ahead and recreate it the same way

Reply:
I believe it is mentioned in HIP. Need to re-check

Reply:
Fixed